content/en/guides/internals-glossary/context.md

```diff
@@ -95,13 +95,23 @@ Boolean to let you know if the method/middleware is called from webpack hot modu
 
 Use this method to redirect the user to another route, the status code is used on the server-side, defaults to `302`. `redirect([status,] path [, query])`.
 
+Examples:
```

Comment:
I guess it'd be good to add the client-side caveat in the paragraph (see https://github.com/nuxt/nuxt.js/issues/4491)

Reply:
You mean this https://github.com/nuxt/nuxt.js/issues/4491#issuecomment-648979464 @manniL ?

Reply:
Yup, exactly